CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.2.0] - 2021-01-28
+## [0.2.0] - 2021-01-29
 
 ### Added
 
```

Comment:
Do not touch previous versions in changelog unless you exactly know why. Version 0.2.0 was released at the 28.01 not the 29.01. Do not do things like this. Every new change needs to be specified under Unreleased section.